src/trix/config/input.js

```diff
@@ -11,16 +13,18 @@ const input = {
       return 0
     }
   },
-  pickFiles(callback) {
-    const input = makeElement("input", { type: "file", multiple: true, hidden: true, id: this.fileInputId })
+  pickFiles(editorController) {
+    const editorElement = editorController.element
+    const acceptTypes = editorElement.getAttribute("trix-attachment-accept") || this.acceptedFileTypes
+    const input = makeElement("input", { type: "file", multiple: true, hidden: true, id: this.fileInputId, accept: acceptTypes })
```

Comment:
Not for this PR specifically, but should we allow customizing whether it allows multiple files or not?